yt_dlp/extractor/youtube.py

```diff
@@ -807,6 +807,12 @@ def _extract_video(self, renderer):
         description = self._get_text(renderer, 'descriptionSnippet')
         duration = parse_duration(self._get_text(
             renderer, 'lengthText', ('thumbnailOverlays', ..., 'thumbnailOverlayTimeStatusRenderer', 'text')))
+        if duration is None:
+            duration = parse_duration(self._search_regex(
+                r'(?i)(ago)(?!.*\1)\s+(?P<duration>[a-z0-9 ,]+?)(?:\s+[\d,]+\s+views)?(?:\s+-\s+play\s+short)?$',
+                traverse_obj(renderer, ('title', 'accessibility', 'accessibilityData', 'label')),
```

Comment:
We need to handle the case that `traverse_obj(renderer, ('title', 'accessibility', 'accessibilityData', 'label'))` returns `None`, which crashes `_search_regex`. This can happen when downloading a playlist that contains unavailable videos.

Reply:
Oh true, thanks for catching that.

Reply:
should be fixed now. thanks